charts/manticoresearch/templates/manticore-balancer.yaml

```diff
@@ -120,6 +127,11 @@ spec:
         - name: config-volume
           configMap:
             name: {{ include "manticoresearch.fullname" . }}-balancer-config
+        {{- range .Values.balancer.volumes }}
+        - name: {{ .name }}
+          configMap:
```

Comment:
Using the config map here is not a good idea. The ConfigMap size limit is 1 MB.

Wordforms can be pretty big, it's [noticed](https://manual.manticoresearch.com/dev/Creating_a_table/NLP_and_tokenization/Wordforms#wordforms) in our documentation

Reply:
I'll look into this.

Reply:
Ok, looked into this. I'm at a loss as how to solve this. Do you have a suggestion for me?